tests/test_wtw.py

```diff
@@ -113,6 +113,26 @@ def test_wwtw_overrides(self):
         self.assertEqual(wwtw.process_parameters['phosphate']['constant'], 0.01)
         self.assertEqual(wwtw.process_parameters['volume']['constant'], vol)
         self.assertEqual(wwtw.stormwater_storage_capacity, 100)
+    
+    def test_fwtw_overrides(self):
+        fwtw = FWTW(name='')
+        vol = fwtw.process_parameters['volume']['constant']
+        fwtw.apply_overrides({'treatment_throughput_capacity' : 20,
+                              'process_parameters' : {'phosphate' : 
+                                                      {'constant' : 0.01}},
```

Comment:
Maybe use something that isn't the default value ;)

```suggestion
                                                      {'constant' : 0.02}},
```